packages/cli/src/commands/setup/deploy/providers/render.js

```diff
@@ -0,0 +1,114 @@
+import fs from 'fs'
+import path from 'path'
+
+import { getPaths } from 'src/lib'
+
+const PROJECT_NAME = getPaths().base.match(/[^/|\\]+$/)[0]
+
+const RENDER_YAML = (database) => {
+  return `services:
+- type: web
+  name: ${PROJECT_NAME}-web
+  env: static
+  buildCommand: yarn rw deploy render web
+  staticPublishPath: ./web/dist
+  envVars:
+  - key: NODE_VERSION
+    value: 14
+  routes:
+  - type: rewrite
+    source: /.redwood/functions/*
+    destination: replace_me_with_api_url/*
+  - type: rewrite
+    source: /*
+    destination: /index.html
+
+- type: web
```

Comment:
```suggestion
- type: api
```

Should be api?

Reply:
No, it does need to be `type: web`. Check out our yaml spec for more information about the types it can be https://render.com/docs/yaml-spec.

Web is more of an omnibus type for all publically accessible services including the Api.